src/Zonotope.jl

```diff
@@ -129,6 +129,24 @@ function center(Z::Zonotope{N})::Vector{N} where {N<:Real}
     return Z.center
 end
 
+"""
+   generators(Z::Zonotope)
+
+Return the generators of a zonotope.
+
+### Input
+
+- `Z` -- zonotope
+
+### Output
+
+The generators of the zonotope.
+"""
+
+
+function generators(Z::Zonotope)
```

Comment:
Remove the empty lines between comment and function.

Reply:
OK, thanks.